README.md

```diff
@@ -1,16 +1,288 @@
+</script>
+```
+
+Next, the FSXAApp can be used in the template, passing different parameters there:
+
+
+| Parameter  | Explanation  |
+|---|---|
+| defaultLocale  |  Provide the locale your content should be displayed in, if no initial path is passed | 
+| devMode  | When activated, you will be shown useful information with which you can start developing your ui.  |     
+| handleRouteChange  | Required callback that will be triggered, when the route should be changed  |     
+| currentPath  | You can specify the path of the page that should be displayed  |     
+| components  |  Mapping of the FirstSpirit components |     
+
+This is what the component looks like:
+
+```vue
+<template>
+  <fsxa-app
+      defaultLocale="de_DE"
+      devMode="true"
+      :handleRouteChange="changeRoute"
+      :currentPath="route"
+      :components="components"
+  />
+</template>
+
+<script>
+import { Component } from "vue-property-decorator";
+import Vue from "vue";
+import { FSXAApp } from "fsxa-pattern-library";
+
+@Component({
+  name: "VueFSXAApp",
+  components: {
+    "fsxa-app": FSXAApp
+  }
+})
+export default class App extends Vue {
+  route = location.pathname;
+
+  onRouteChange() {
+    this.route = location.pathname;
+  }
+
+  changeRoute(route) {
+    history.pushState(null, "Title", route);
+    this.route = route;
+  }
+
+  get components() {
+    return {};
+  }
+}
+</script>
+```
+
+### Nuxt.js
+
+To use the FSXA-Pattern-Library in a Nuxt project, the [FSXA-Nuxt-Module](https://github.com/e-Spirit/fsxa-nuxt-module) has to be installed with:
+`npm install fsxa-nuxt-module`.
+
+In the `nuxt.config.ts` file this module must be included.
+```typescript
+buildModules: [
+    'fsxa-nuxt-module',
+  ],
+```
+
+In addition, the CSS of the FSXA-Pattern-Library must be included in the `nuxt.config.ts` file:
+```typescript
+css: ['fsxa-pattern-library/dist/fsxa-pattern-library.css']
+```
+
+After that, the `.env` file must be created in the root of the project and the environment variables must be set there:
+```dotenv
+# APIKey used for authentication against the CaaS
+FSXA_API_KEY=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+# URL pointing to your CaaS Instance
+FSXA_CAAS=https://url-to.your.caas
+# ID of your project
+FSXA_PROJECT_ID=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+# URL pointing to the NavigationService
+FSXA_NAVIGATION_SERVICE=https://your.navigation-service.cloud/navigation
+# Which mode should be used? (preview/release)
+FSXA_MODE=preview
+# KEY:VALUE map where semicolon is used as separator (key:uuid;key:uuid)
+FSXA_REMOTES=en_EN
+# This tenantId is required, when using caas-connect module 3.0.9 and above
+FSXA_TENANT_ID=xxxxxxxxxxxxxxx
+```
+
+Next, the `fsxa.config.ts` must be created:
+```typescript
+export default {
+  devMode: false,
+  defaultLocale: 'de_DE'
+}
+```
+
+Here the `devMode` is specified. the default language and under which path the components are specified.
+Additionally, `customRoutes` and `components` can be provided.
+
```

Comment:
I would formulate it something like this: In order to correctly load your components you can either choose to add the paths to the configuration -> link to configuration section with the example, or you can choose to follow convention by placing your components in specific folders -> link to convention section with the folder structure